rotkehlchen/premium/premium.py

```diff
@@ -176,10 +178,13 @@ def __init__(self, credentials: PremiumCredentials):
             allowed_methods=False,  # retry on all method verbs
         )
         self.session.mount('https://', adapter)
+        self.session.verify = False
         self.apiversion = '1'
-        self.rotki_api = f'https://rotki.com/api/{self.apiversion}/'
-        self.rotki_nest = f'https://rotki.com/nest/{self.apiversion}/'
+        self.rotki_api = f'https://localhost/api/{self.apiversion}/'
+        self.rotki_web = f'https://localhost/webapi/{self.apiversion}/'
```

Comment:
Did we agree on this url for that? What was the latest talk with kelsos?

Reply:
This is my error since I made this change last week to test and forgot to revert the values used for testing